nautobot_device_lifecycle_mgmt/filters.py

```diff
@@ -74,25 +74,17 @@ class Meta:
 
         model = HardwareLCM
 
-        fields = [
-            "end_of_sale",
-            "end_of_support",
-            "end_of_sw_releases",
-            "end_of_security_patches",
-            "inventory_item",
-            "documentation_url",
-            "expired",
-        ]
-
-    def search(self, queryset, name, value):  # pylint: disable=unused-argument, no-self-use
+        fields = '__all__'
```

Comment:
the `ruff-format` ci step should be failing on these because they should be `"__all__"`. We'll need to figure out why it's not failing and fix it.

Reply:
https://github.com/nautobot/cookiecutter-nautobot-app/issues/160